bot.py

```diff
@@ -2935,6 +3015,14 @@ def sort_roles(role_list):
                     await send_lobby(random.choice(lang['hastotems']).format('**, **'.join([get_name(x) for x in totem_holders[:-1]]), get_name(totem_holders[-1])))
 
             for player in killed_temp:
+                lover = o.split(':')[1]
+                if lover in killed_temp and player in killed_temp:
+                    #fix for lover suicide message appears if player dies even tho lover died already
+                    session[1][player][4].remove('lover:' + lover)
+                    session[1][lover][4].remove('lover:' + player)
+                    await player_death(player, 'night kill')
```

Comment:
this only happens if both lovers were killed at night, and that is a fix for this 
![tempae](https://user-images.githubusercontent.com/30117228/29088134-1fb73898-7c70-11e7-8d11-9f361dbac112.PNG)